kaldb/src/test/java/com/slack/kaldb/chunk/ReadOnlyChunkImplTest.java

```diff
@@ -55,17 +55,17 @@
 import org.junit.ClassRule;
 import org.junit.Test;
 import software.amazon.awssdk.services.s3.S3Client;
-import software.amazon.awssdk.services.s3.model.CreateBucketRequest;
 
 public class ReadOnlyChunkImplTest {
-  private final String TEST_S3_BUCKET =
-      String.format("%sBucket", this.getClass().getSimpleName()).toLowerCase();
+  private static final String TEST_S3_BUCKET = "read-only-chunk-impl-test";
 
   private TestingServer testingServer;
   private MeterRegistry meterRegistry;
   private S3BlobFs s3BlobFs;
 
-  @ClassRule public static final S3MockRule S3_MOCK_RULE = S3MockRule.builder().silent().build();
+  @ClassRule
+  public static final S3MockRule S3_MOCK_RULE =
```

Comment:
This is due to a change in Adobe S3Mock 2.5 that started validating against duplicate bucket names - https://github.com/adobe/S3Mock/blob/main/CHANGELOG.md#250. Since we used a class rule but tried to init the same bucket in `@Each`, this would cause subsequent tests to fail.